mmpy_bot/plugins/example.py

```diff
@@ -110,7 +119,13 @@ async def ping_reply(self, message: Message):
         """Pong."""
         self.driver.reply_to(message, "pong")
 
-    @listen_to("^reply at (.*)$", re.IGNORECASE, needs_mention=True)
+    @listen_to(
+        "^reply at (.*)$",
+        re.IGNORECASE,
+        needs_mention=True,
+        category="schedule",
+        human_description="reply at TIMESTAMP",
+    )
```

Comment:
I understand that you want this info for the `HelpPlugin`, but I don't like how verbose this has made the `ExamplePlugin`. The `ExamplePlugin` serves as a showcase of how easily you can create various kinds of behaviour, and the massive decorators with optional additional information take away from that purpose a bit... Are you okay with removing this for most of these functions and instead just relying on the documentation to explain the help behaviour?